main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAgencySchema.java

```diff
@@ -34,6 +29,7 @@ public interface GtfsAgencySchema extends GtfsEntity {
   String agencyId();
 
   @Required
+  @MixedCase
```

Comment:
I know this annotation is intended for more than this field, but it's going to be interesting to see how this actually validates for agency name.  For example, if the agency name is "SFMTA" or "MARTA" (aka an acronym) it won't actually be mixed-case.